src/firekit.ts

```diff
@@ -1884,14 +1888,74 @@ export class RoarFirekit {
       throw new Error('Grade cannot be empty.');
     }
 
+    // +------------------------------------------------------------------+
+    // | Split out org updates, user record updates, and user doc updates |
+    // +------------------------------------------------------------------+
+    const { addOrgs, removeOrgs, password, ...sendUserData } = userData;
+    const email = _get(sendUserData, 'email', null);
+
+    // If orgs are supplied, update user's admin orgs.
+    if (!_isEmpty(addOrgs)) {
+      const sendAddOrgs = {
+        requesterUid,
+        targetUid: id,
+        ...(addOrgs as {
+          districts: string[];
+          schools: string[];
+          classes: string[];
+          groups: string[];
+          families: string[];
+        }),
+      };
+      const cloudAppendToAdminClaims = httpsCallable(this.admin!.functions, 'appendToAdminClaims');
+      const appendResponse = await cloudAppendToAdminClaims(sendAddOrgs);
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      if (_get(appendResponse.data as any, 'status') !== 'ok') {
+        throw new Error('Failed to append to admin claims.');
+      }
+    }
+    if (!_isEmpty(removeOrgs)) {
+      const sendRemoveOrgs = {
+        requesterUid,
+        targetUid: id,
+        ...(removeOrgs as {
+          districts: string[];
+          schools: string[];
+          classes: string[];
+          groups: string[];
+          families: string[];
+        }),
+      };
+      const cloudRemoveFromAdminClaims = httpsCallable(this.admin!.functions, 'removeFromAdminClaims');
+      const appendResponse = await cloudRemoveFromAdminClaims(sendRemoveOrgs);
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      if (_get(appendResponse.data as any, 'status') !== 'ok') {
+        throw new Error('Failed to append to admin orgs.');
+      }
+    }
+
```

Comment:
```suggestion
await this._updateAdminOrgs(requesterUid, id, addOrgs as Partial<AdminOrgs>, 'appendToAdminClaims');
await this._updateAdminOrgs(requesterUid, id, removeOrgs as Partial<AdminOrgs>, 'removeFromAdminClaims');
```
Then call the new methods here.